app/Nova/Skill.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Nova;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\BelongsTo;
+use Laravel\Nova\Fields\BelongsToMany;
+use Laravel\Nova\Fields\Boolean;
+use Laravel\Nova\Fields\ID;
+use MrMonat\Translatable\Translatable;
+
+class Skill extends Resource
+{
+
+    public static $model = 'App\Skill';
+
+    public static $title = 'id';
+
+    public static $search = [
+        'id', 'name'
```

Comment:
I'm guessing people won't search the id, so we can probably drop that
```suggestion
        name'
```